packages/js/onboarding/src/components/WCPayBanner/WCPayBanner.scss

```diff
@@ -1,6 +1,7 @@
 .woocommerce-recommended-payments-banner {
 	margin: 0 15px 10px 0;
 	animation: isLoaded;
+	min-width: 550px;
```

Comment:
Noting that this was earlier `750px` before being removed in https://github.com/woocommerce/woocommerce/pull/46003/files#diff-7b7f0b500af945ff65443ee6f1dfb5e54091a530ff2fbd1b7886724328f529fe. I haven't manually tested this branch, but it seems 550 is enough?

Reply:
Indeed, it was initially set to `750px`, but based on my tests, it pushes the banner beyond the visible area on iPhone SE. `550px` seems to be the optimal size that accommodates all viewports.